src/components/commentPost.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useContext } from "react"
+import { UserContext } from "../contexts/userContext"
+import { getCommentById, postComment } from "./api"
+
+export const CommentForm = ({ comments, setComments, article_id }) => {
+    const [commentText, setCommentText] = useState("")
+    const [isLoading, setIsLoading] = useState(false)
+    const {defaultUser: {username}} = useContext(UserContext)
+
+    const handleChange = ({target:{value}}) =>{
+        setCommentText(value)
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        const newComment = {username: username, body: commentText}
+        setIsLoading(true)
+        setCommentText("")
+    
+
+    postComment(newComment, article_id).then((comment)=>{
```

Comment:
This is going to need a catch block. If I try and post while offline I'm stuck in what looks like a very slow success state. Let the user know this hasn't worked properly 👍 This can be done on the error handling ticket though 😄 
![image](https://github.com/user-attachments/assets/97508e54-a87b-4346-a962-b6c480842801)